### ISSUE TITLE
Dungeon/Raid bugs

### ISSUE THREAD

**Issue by cybrann** [cooler-SAI/MopCore547#34]
Report any dungeon or raid bug you find here.

**Comment by alexkulya** [comment]
Any progress?

**Comment by cybrann** [comment]
Yes, I'm doing some fixes. It will be pushed whenever I think they are ready.

**Comment by alexkulya** [comment]
I think this past week or month to test all the dungeons and raids of MOP content and I will write here about the results.

**Comment by alexkulya** [comment]
Not a whole lot of time. Now I actively involved in developing Draenor and WotLK Blizzlike.

**Comment by phulo** [comment]
i can compile the code on linux system?

**Comment by phulo** [comment]
it seem the code is death.. NO updates

**Comment by cybrann** [comment]
It will get updated, but I do not know what to fix. I need bug reports.

**Comment by alexkulya** [comment]
Updated. Add Cataclysm content.

**Comment by phulo** [comment]
the code do not compile on linux system why? someone can test it?

**Comment by cooler-SAI** [comment]
@phulo , no linux support

**Comment by neltharionorg** [comment]
> no linux support

@cooler-SAI why?

**Comment by cooler-SAI** [comment]
coz we don't know it

**Comment by cybrann** [comment]
If you guys would be kind to start posting some bugs here, I will be able to do some work. If no bugs reported I cannot do any work on the cata/mop dungeons/raids.

**Comment by Crypticaz** [comment]
Temple of the jade forest - Wise Mari doesn't do anything like he is supposed to. He doesn't put a bubble around himself to make himself immune to everything, he doesn't cast spells and he doesn't summon the water elementals.

**Comment by Dieguitox87** [comment]
After Sha of Pride in Siege of Orgrimmar you port into a boat. Nothing happens after. It should take you into the shore

**Comment by Dieguitox87** [comment]
When you engage sha of doubt inside of the jade serpent it makes the server crash.
